resque-scheduler.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest'
   spec.add_development_dependency 'mocha'
   spec.add_development_dependency 'pry'
+  spec.add_development_dependency 'byebug'
```

Comment:
I think byebug is MRI specific, so this should probably be put in the Gemfile as `gem 'byebug', :platforms => :mri`